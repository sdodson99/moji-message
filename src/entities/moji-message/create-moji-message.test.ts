import { createMojiMessage, CreateMojiMessageRequest } from '.';

describe('createMojiMessage', () => {
  let request: CreateMojiMessageRequest;

  beforeEach(() => {
    request = {
      message: 'abcdefghijklmnopqrstuvwxyz ?! 1234567890',
      messageEmoji: '🔴',
      backgroundEmoji: '⚪️',
    };
  });

  it('returns encoded message', () => {
    const mojiMessage = createMojiMessage(request);

    expect(mojiMessage).toMatchSnapshot();
  });

  it('encodes message with emoji', () => {
    const mojiMessage = createMojiMessage(request);

    expect(mojiMessage).toContain(request.messageEmoji);
  });

  it('encodes message background with background emoji', () => {
    const mojiMessage = createMojiMessage(request);

    expect(mojiMessage).toContain(request.backgroundEmoji);
  });

  it('returns empty message for unknown characters', () => {
    request.message = '%@#$';

    const mojiMessage = createMojiMessage(request);

    expect(mojiMessage).toBeFalsy();
  });
});
