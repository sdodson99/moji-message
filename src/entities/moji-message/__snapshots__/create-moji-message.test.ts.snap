// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`createMojiMessage > returns encoded message 1`] = `
"⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️🔴⚪️⚪️
⚪️🔴⚪️🔴⚪️⚪️⚪️
⚪️🔴🔴⚪️⚪️⚪️⚪️
⚪️🔴⚪️🔴⚪️⚪️⚪️
⚪️🔴⚪️⚪️🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴⚪️🔴🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴⚪️⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️⚪️🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️🔴🔴⚪️
⚪️⚪️🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️🔴⚪️⚪️⚪️
⚪️🔴⚪️⚪️🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴🔴⚪️🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴🔴⚪️⚪️⚪️
⚪️🔴⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️🔴🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️⚪️🔴⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️⚪️⚪️⚪️
⚪️🔴⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴🔴⚪️
⚪️⚪️⚪️⚪️🔴⚪️⚪️
⚪️⚪️⚪️🔴⚪️⚪️⚪️
⚪️⚪️🔴⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️🔴🔴⚪️
⚪️🔴⚪️🔴⚪️🔴⚪️
⚪️🔴🔴⚪️⚪️🔴⚪️
⚪️🔴⚪️⚪️⚪️🔴⚪️
⚪️⚪️🔴🔴🔴⚪️⚪️
⚪️⚪️⚪️⚪️⚪️⚪️⚪️"
`;
